__tests__/src/containers/DataPackageViewContainer.test.js

```diff
@@ -73,7 +65,7 @@ const mockData = [
 
 describe("Datapackage View Container", () => {
 
-  const wrapper = shallow(<DataPackageViewContainer />);
+  const wrapper = shallow(<DataPackageView publisherName="publisher" packageName="pack" metadata={{readme: ''}}/>);
```

Comment:
Did you launch tests? This one should fail. You removed "metadata" from DataPackageView props, as we discussed.
Can you please enable travis before merging this?